cfme/fixtures/networks.py

```diff
@@ -1,17 +1,8 @@
-import re
-from argparse import Namespace
-from contextlib import contextmanager
-
 import pytest
 
-from cfme.exceptions import NeedleNotFoundInLog
-from cfme.utils.log import logger
-from cfme.utils.wait import TimedOutError
-from cfme.utils.wait import wait_for
-
 
 @pytest.fixture
-def targeted_refresh(merkyl_setup, merkyl_inspector):
+def targeted_refresh():
```

Comment:
I see that `test_events_are_triggered.py` still uses this fixture. Should we update it for that test case/module?

Reply:
it's still used by several nuage tests. we don't run those tests due to absent provider. plus I doubt someone runs those at all. however, I don't want to remove those tests since I haven't added them and I'm not sure whether we are going to run those tests somewhere.. That's why I removed fixtures related to merkyl and left the rest.